side_little: [0.126928, 0.226531, 0.521694, 0.719380, -0.118530,  -0.239714,  0.054018, -0.273228,  -0.162564,  -0.141151,  -0.206231,  -0.128360,  -0.102260,  -0.101524,  -0.051255]

frontal_little: [0.007551,  -0.008612,  -0.084668,  -0.089635,  -0.036814,  0.074018, 0.078178, 0.064221, 0.014458, -0.004616,  0.024618, 0.003117, 0.004746, -0.021997,  -0.003253]

vertical_little: [-0.003779,  0.072345, -0.002825,  -0.340218,  -0.202082,  0.030993, 0.058855, -0.022684,  0.016284, 0.048129, 0.143108, 0.105347, 0.073801, 0.060821, 0.017264]

frontal_ring: [-0.031323, -0.107100,  -0.101094,  -0.068674,  -0.037239,  -0.054615,  0.004176, 0.002501, 0.002791, 0.010981, 0.037363, 0.068047, 0.095295, 0.110739, 0.076166]

vertical_ring: [-0.005350,  0.012427, -0.112518,  -0.225242,  -0.118230,  -0.116978,  -0.119305,  0.048908, -0.007523,  0.058943, 0.054863, 0.076456, 0.091207, 0.160873, 0.106235]

frontal_middle: [-0.008128, -0.014458,  -0.028806,  -0.102790,  -0.047698,  0.051575, 0.032624, -0.000422,  0.020882, 0.023808, 0.043664, 0.024263, 0.020502, 0.013470, 0.001316]

vertical_middle: [0.043086, -0.026955,  -0.212838,  -0.323673,  -0.256955,  -0.066615,  0.094044, 0.175965, 0.151017, 0.080638, 0.026114, 0.029692, 0.018593, 0.039212, 0.104540]

side_index: [-0.045438, 0.063278, -0.132459,  -0.329795,  0.096313, 0.279608, 0.187720, 0.068596, 0.220162, 0.152750, 0.019130, 0.134549, 0.082376, 0.074717, 0.026645]

frontal_index: [-0.000762,  -0.016201,  -0.105782,  -0.010287,  0.035482, 0.083347, -0.032025,  -0.008275,  0.075906, 0.024273, 0.141742, 0.108033, 0.091355, 0.049766, 0.066505]

vertical_index: [-0.003779, 0.072345, -0.002825,  -0.340218,  -0.202082,  0.030993, 0.058855, -0.022684,  0.016284, 0.048129, 0.143108, 0.105347, 0.073801, 0.060821, 0.017264]

side_thumb: [-0.036627, -0.021502,  -0.003647,  -0.038065,  -0.133008,  -0.086265,  -0.000901,  0.058743, 0.063317, 0.052453, 0.080114, 0.090317, 0.075165, 0.009506, -0.039639]

frontal_thumb: [0.056385, -0.018360,  -0.060231,  0.089327, 0.052637, 0.085123, 0.031455, 0.022489, 0.003034, 0.029154, 0.034337, 0.034599, 0.013380, 0.002979, 0.037891]  

vertical_thumb: [-0.150252, -0.234870,  -0.170568,  -0.513898,  -0.224666,  0.036356, 0.208052, 0.214301, 0.153591, 0.173598, 0.169599, 0.183140, 0.124223, 0.047572, 0.025727]